code/modules/admin/sql_notes.dm

```diff
@@ -52,8 +54,8 @@
 		log_game("SQL ERROR adding new note to table. Error : \[[err]\]\n")
 		return
 	if(logged)
-		log_admin("[key_name(usr)] has added a note to [target_ckey]: [notetext]")
-		message_admins("[key_name_admin(usr)] has added a note to [target_ckey]:<br>[notetext]")
+		log_admin("[usr ? key_name(usr) : "Bot" ] has added a note to [target_ckey]: [notetext]")
```

Comment:
Can perhaps use the `adminckey` here instead of `"Bot"`. It will be filled with `"Automated Ban"` for the auto bans. Also allows other sources to be recognized more easily